docs/scripts/formattedTSDemos.js

```diff
@@ -90,15 +89,18 @@ const previewOverride = {
 async function transpileFile(tsxPath, program, ignoreCache = false) {
   const jsPath = tsxPath.replace(/\.tsx?$/, '.js');
   try {
-    if (!ignoreCache && (await fse.exists(jsPath))) {
-      const [jsStat, tsxStat] = await Promise.all([fse.stat(jsPath), fse.stat(tsxPath)]);
+    if (!ignoreCache && (await fs.promises.exists(jsPath))) {
```

Comment:
This line introduced an issue. Fix [here](https://github.com/mui/mui-x/pull/19132)